master/getting-started/bare-metal/bare-metal.md

```diff
@@ -31,21 +32,16 @@ However, for host endpoints, Calico is more lenient; it only polices
 traffic to/from interfaces that it's been explicitly told about. Traffic
 to/from other interfaces is left alone.
 
-As of Calico v2.1.0, Calico applies host endpoint security policy both to traffic
-that is terminated locally, and to traffic that is forwarded between host
-endpoints.  Previously, policy was only applied to traffic that was terminated
-locally.  The change allows Calico to be used to secure a NAT gateway or router.
-Calico supports selector-based policy as normal when running on a gateway or router
-allowing for rich, dynamic security policy based on the labels attached to your
-workloads.
-
-> **Note**: If you have a host with workloads on it then traffic that is forwarded to
-> workloads bypasses the policy applied to host endpoints. If that weren't the
-> case, the host endpoint policy would need to be very broad to allow all
-> traffic destined for any possible workload.
+Host endpoint policy allows Calico to be used to secure a NAT gateway or router. Calico supports selector-based policy as normal when running on a gateway or router allowing for rich, dynamic security policy based on the labels attached to your
+workloads. Host endpoint policy applies to three types of traffic, traffic
+that is terminated locally, traffic that is forwarded between host endpoints, and traffic that is forwarded between host endpoint and workload endpoint on the same host. Users can specify if a given host endpoint policy applies to forwarded traffic or not by setting `applyOnForward` flag. 
```

Comment:
This is really nice, I would just suggest breaking it into bullet points to make it easier to follow. So, it would be something like: 
```
Host endpoint policy applies to three types of traffic:
- Traffic that is terminated locally
- Traffic that is forwarded between host endpoints
- Traffic that is forwarded between endpoints on the same host
```
I deleted "host" and "workload" from the third bullet because it is a little confusing to have two hosts in this bullet. I am not sure, can there be more than one host endpoint on a single host? Still learning so feel free to use your best judgement here.

Reply:
I am not sure, but I think that this feature gets turned on and off via `applyOnForward`. If so, I would recommend changing the intro sentence to "You can apply host endpoint policies to three types of traffic:"

Reply:
Yes. There could be more than one host endpoint on a single host.